common/PlatformTypeUtils.gd

```diff
@@ -0,0 +1,26 @@
+extends Node
+
```

Comment:
This should use `class_name PlatformTypeUtils` here instead of adding it to autoloads

Reply:
Oh, yes I keep forgetting about that!  Will do!